<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>button_back_2</class>
 <widget class="QDialog" name="button_back_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1648</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_pasivna">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pasivna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_agresivna">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agresivna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_back_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NAZAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_glhf">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>590</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GLHF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
